port/webhook/schema.go

```diff
@@ -50,6 +50,10 @@ func WebhookMappingSchema() map[string]schema.Attribute {
 			MarkdownDescription: "The items to parser of the mapping",
 			Optional:            true,
 		},
+		"operation": schema.StringAttribute{
```

Comment:
This should be an explicit enum (`create` | `delete`) according to https://github.com/port-labs/Port/blob/main/apps/port-api/src/api/routes/v1/schemas/webhookIntegration/patchIntegrationRequestBodySchema.ts#L93